lib/ex_aws/sns/public_key_cache.ex

```diff
@@ -0,0 +1,59 @@
+defmodule ExAws.SNS.PublicKeyCache do
```

Comment:
SNS certificates are valid for 1year.
<img width="524" alt="sns_amazonaws_com" src="https://cloud.githubusercontent.com/assets/3340262/20256778/95f24880-aa80-11e6-9322-0b95a963fedf.png">